dspace/config/spring/api/core-services.xml

```diff
@@ -143,70 +148,206 @@
+         Since the journal.identifier.issn metadata field can't be filled in through the metadata of the journalvolume
+         alone, we need to pass this call higher up the chain to the journalvolume's relationships to find the journal.
+         Therefore we've created this Related bean which will look at the journalvolume's relationships with type
+         'isJournalOfVolume' to find the journal that this journalvolume is related to and it'll then call the next
+         VirtualBean with that journal to retrieve the value. This will leave the retrieving of the values to the
+         next bean in the chain -->
+    <bean class="org.dspace.content.virtual.Related" id="volumeJournal_issn_related">
+        <property name="relationshipTypeString" value="isJournalOfVolume"/>
+        <property name="place" value="1"/>
+        <property name="virtualBean" ref="volumeJournal_issn"/>
+    </bean>
 
     <util:map id="isJournalOfVolumeMap">
-        <entry key="journal_identifier_issn" value-ref="volumeJournal_issn"/>
-        <entry key="journal_title" value-ref="volumeJournal_title"/>
+        <entry key="journal.identifier.issn" value-ref="volumeJournal_issn"/>
+        <entry key="journal.title" value-ref="volumeJournal_title"/>
     </util:map>
-    <util:list id="volumeJournal_issn">
-        <value>journal.identifier.issn</value>
-    </util:list>
-    <util:list id="volumeJournal_title">
-        <value>journal.identifier.name</value>
-    </util:list>
-	
+    <bean class="org.dspace.content.virtual.Collected" id="volumeJournal_issn">
+        <property name="fields">
+            <util:list>
+                <value>journal.identifier.issn</value>
+            </util:list>
+        </property>
+    </bean>
+    <bean class="org.dspace.content.virtual.Collected" id="volumeJournal_title">
+        <property name="fields">
+            <util:list>
+                <value>journal.identifier.name</value>
+            </util:list>
+        </property>
+    </bean>
+
     <util:map id="isVolumeOfJournalMap">
     </util:map>
 	
     <util:map id="isIssueOfJournalVolumeMap">
         <entry key="journalissue_identifier_number" value-ref="journalIssue_number"/>
     </util:map>
-    <util:list id="journalIssue_number">
-        <value>journalissue.identifier.number</value>
-    </util:list>
+    <bean class="org.dspace.content.virtual.Collected" id="journalIssue_number">
+        <property name="fields">
+            <util:list>
+                <value>journalissue.identifier.number</value>
+            </util:list>
+        </property>
+    </bean>
+
+    <util:map id="isJournalIssueOfPublicationMap">
+        <entry key="journal.identifier.issn"  value-ref="issueVolumeJournal_issn_related"/>
+    </util:map>
+
+    <!-- This is a Related bean, which will find the relationshiptype defined in the relationshipTypeString property on
```

Comment:
Similar to above comment, it'd be good to copy this detailed example into the `Related` bean's Javadocs. Again, we likely don't need detailed examples for every instance of `Related` in this config file, as long as we can link to Javadocs for more info.

Reply:
I've moved the general comments to the JavaDoc of the class, and only left the specific comments explaining the use case for that configuration